### PR TITLE
fixed some code

### PR DESCRIPTION
**Pull request by 4lxm4x**
Hello. Moja popytka nomer 5

### CODE REVIEW

File: src/js/02-video.js [hunk 1]
```diff
@@ -1,11 +1,9 @@
 import Vimeo from '@vimeo/player';
-import _ from 'lodash';
+import throttle from 'lodash';
```
Comment (murarova) [inline src/js/02-video.js]: the import has to be like:
`import throttle from 'lodash/throttle';`

and then you can use it without _
`throttle()`

File: src/js/03-feedback.js [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+import throttle from 'lodash';
```
Comment (murarova) [inline src/js/03-feedback.js]: the same import fix

File: src/js/02-video.js (hunk 1 above)
Comment (4lxm4x) [reply src/js/02-video.js]: done

File: src/js/03-feedback.js (hunk 2 above)
Comment (4lxm4x) [reply src/js/03-feedback.js]: done

**Review by murarova — APPROVED** [review]
nice

**Pull request merged**